stories/src/csv.stories.tsx

```diff
@@ -39,7 +39,9 @@ storiesOf('CSV from Flux', module)
         'MM/DD/YYYY HH:mm:ss.sss': 'MM/DD/YYYY HH:mm:ss.sss',
         'YYYY/MM/DD HH:mm:ss': 'YYYY/MM/DD HH:mm:ss',
         'YYYY-MM-DD HH:mm:ss ZZ': 'YYYY-MM-DD HH:mm:ss ZZ',
+        'YYYY-MM-DD HH:mm:ss a ZZ': 'YYYY-MM-DD HH:mm:ss a ZZ',
```

Comment:
Is it possible to specify ‘a’ with ‘ZZ’?

Reply:
why would you not be able to do this? ZZ says the timezone is UTC

Reply:
This is a storybook change, which is really for testing. I wanted to see what happens if both UTC and am/pm were given in the format by the user. It is for edge case testing. I don't think a serious user would send those options in.

And for your curiosity, am/pm always wins. If 'a' is present, am/pm forces itself into the format regardless of the time zone.

Reply:
TIL. Thanks

Reply:
>Practically speaking, if a user in the US toggled the UTC timezone, the format of the times shouldn’t have the option of being am/pm

I don't follow why not. UTC and AM/PM are on different axes - they're not mutually exclusive, even though we can't imagine it as a common use case. AM/PM is mutually exclusive with military / 24 hour time, because you can't have 13:15 PM. But as a user in the US, I might want to see my data relative to UTC timezone, but in my American style, so I'd rather see 3 PM than 1500. The timezone of the date shouldn't affect displaying am/pm.

I think the issue here isn't the `ZZ`, it's the `HH`, which renders the time in a 0 padded 24 hour format, which is weird with am / pm